bindgen-cli/options.rs

```diff
@@ -504,8 +508,9 @@ where
         with_derive_custom_enum,
         with_derive_custom_union,
         wrap_static_fns,
-        wrap_static_fns_path,
+        native_code_generation_path,
```

Comment:
I don't love the `native` here, maybe just saying `c_code_generation_path` or `wrapper_code_generation_path` is better?